LondonBritishLibrary/orient/BLorient687.xml

```diff
@@ -37,6 +42,507 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                <title type="complete" ref="LIT4829Martyr"/>
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.29">
+                                <locus from="131ra" to="149vc"/>
+                                <title type="complete" ref="LIT6482GadlaArs"/>
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.30">
+                                <locus from="150ra" to="157rc"/>
+                                <title type="complete" ref="LIT4842Dersan"/>
+                                <textLang mainLang="gez"/>
+                                <note>For <date ref="ethiocal:Tahsas8">8 Tāḫśāś</date>.</note>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.31">
+                                <locus from="157va" to="159vc"/>
+                                <title type="complete" ref="LIT4509GadlaBarb"/>
+                                <textLang mainLang="gez"/>
+                                <note>For <date ref="ethiocal:Tahsas8">8 Tāḫśāś</date>.</note>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.32">
+                                <locus from="160ra" to="161vc"/>
+                                <title type="complete" ref="LIT4510GadlaBa"/>
+                                <textLang mainLang="gez"/>
+                                <note>For <date ref="ethiocal:Tahsas9">9 Tāḫśāś</date>.</note>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.33">
+                                <locus from="162ra" to="162vc"/>
+                                <title type="complete" ref="LIT6500Gadl"/>
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.34">
+                                <locus from="163ra" to="169vc"/>
+                                <title type="complete" ref="LIT6501Gadlu"/>
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.35">
+                                <locus from="170ra" to="175vc"/>
+                                <title type="complete" ref="LIT6505Gadl"/>
+                                <textLang mainLang="gez"/>
+                                
+                                <msItem xml:id="ms_i1.35.1">
+                                    <locus from="170ra" to="172vc"/>
+                                    <title type="complete" ref="LIT6505Gadl#MarmehnamGadl"/>
```

Comment:
I guess here too we would need separate IDs for the Dossier, for the Gadl and for the Miracles, as they may appear independently. I would keep 6505 for the Gadl and add Dossier and Miracles